backend/tests/core/test_security.py

```
@@ -185,15 +192,19 @@ def test_verify_token_without_username(self):
 class TestAuthenticateUser:
     """Test user authentication function"""
 
-    def test_authenticate_user_with_valid_credentials(self, test_db: Session, test_user: User):
+    def test_authenticate_user_with_valid_credentials(
+        self, test_db: Session, test_user: User
```

Ruff (0.14.11)

Unused method argument: `test_user`

(ARG002)